templates/inspec/examples/attributes/google_compute_ssl_policies.erb

```diff
@@ -0,0 +1,2 @@
+project_name = attribute('project_name', default: '<%= grab_attributes['project_name'] -%>')
```

Comment:
Do we need attributes for the singular and plural resources? Will they always be the same?

Reply:
They may be, but not always. We could have multiple tests for a singular resource, each with different attributes, and the plural test could need all the attributes. 

I will probably make attributes more programmatic down the line